modules/navigation/nav_utils.h

```diff
@@ -115,7 +121,7 @@ struct NavigationPoly {
 
 	/// Those 4 variables are used to travel the path backwards.
 	int back_navigation_poly_id = -1;
-	uint32_t back_navigation_edge = UINT32_MAX;
+	int back_navigation_edge = -1;
 	Vector3 back_navigation_edge_pathway_start;
```

Comment:
What was reasoning for changing this to int out of interest?

Reply:
I'm using `-1` to signal to `get_path` that this polygon is part of a link and the midpoints of the edge should not be added to the path.  Instead, only the end points are.